miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/controller/view/EditProjectController.java

```diff
@@ -92,6 +95,8 @@ private ModelAndView setupForm(Project project, ModelMap model) throws IOExcepti
     model.put("project", project);
     model.put("projectDto", mapper.writeValueAsString(Dtos.asDto(project, true)));
 
+    model.put("deliverables", mapper.writeValueAsString(deliverableService.list()));
```

Comment:
* Put this into `formConfig` (below) here so you don't need to do it in the JSP
* Convert to DTOs

You can use `MisoWebUtils#addJsonArray` for this